spec/integ/matrix-to-irc.spec.js

```diff
@@ -240,6 +240,165 @@ describe("Matrix-to-IRC message bridging", function() {
         });
     });
 
+    it("should bridge matrix replies as roughly formatted text", function(done) {
+        // Trigger an original event
+        env.mockAppService._trigger("type:m.room.message", {
+            content: {
+                body: "This is the real message",
+                msgtype: "m.text"
+            },
+            user_id: testUser.id,
+            room_id: roomMapping.roomId,
+            sender: "@friend:bar.com",
+            event_id: "$original:bar.com",
+            type: "m.room.message"
+        }).then(() => {
+            env.ircMock._whenClient(roomMapping.server, testUser.nick, "say",
+            function(client, channel, text) {
+                expect(client.nick).toEqual(testUser.nick);
+                expect(client.addr).toEqual(roomMapping.server);
+                expect(channel).toEqual(roomMapping.channel);
+                expect(text).toEqual('<friend "This is the real message"> Reply Text');
+                done();
+            });
+
+            env.mockAppService._trigger("type:m.room.message", {
+                content: {
+                    body: "> <@somedude:bar.com> This is the fake message\n\nReply Text",
+                    msgtype: "m.text",
+                    "m.relates_to": {
+                        "m.in_reply_to": {
+                          "event_id": "$original:bar.com"
+                        }
+                    },
+                },
+                user_id: testUser.id,
+                room_id: roomMapping.roomId,
+                type: "m.room.message"
+            });
+        });
+    });
+
+    it("should bridge matrix replies as roughly formatted text, newline edition", function(done) {
+       // Trigger an original event
+       env.mockAppService._trigger("type:m.room.message", {
+            content: {
+                body: "\nThis\n is the real message",
+                msgtype: "m.text"
+            },
+            user_id: testUser.id,
+            room_id: roomMapping.roomId,
+            sender: "@friend:bar.com",
+            event_id: "$original:bar.com",
+            type: "m.room.message"
+        }).then(() => {
+            env.ircMock._whenClient(roomMapping.server, testUser.nick, "say",
+            function(client, channel, text) {
+                expect(client.nick).toEqual(testUser.nick);
+                expect(client.addr).toEqual(roomMapping.server);
+                expect(channel).toEqual(roomMapping.channel);
+                expect(text).toEqual('<friend "This"> Reply Text');
+                done();
+            });
+
+            env.mockAppService._trigger("type:m.room.message", {
+                content: {
+                    body: "> <@somedude:bar.com> This is the fake message\n\nReply Text",
+                    msgtype: "m.text",
+                    "m.relates_to": {
```

Comment:
technically this event isn't spec compliant because it doesn't populate the html component (which through this review I'm starting to think is a better idea to use anyways). Spec coming Soon™️

Reply:
applies to other tests too